custom/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
   compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api")
   compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-tooling")
   compileOnly("com.fasterxml.jackson.core:jackson-core:${versions.jackson}")
+  implementation("com.fasterxml.jackson.core:jackson-databind:${versions.jackson}")
```

Comment:
Are you sure `compileOnly` is not good enough here?

Reply:
Yes. it doesn't work with compileOnly